magma/tests/lib.rs

```diff
@@ -23,3 +30,30 @@ fn magma_test() {
     state.decrypt_block(&mut block);
     assert_eq!(&plaintext, block.as_slice());
 }
+
+/*
+// an attempt to test with vectors from
+// https://github.com/gost-engine/engine/blob/master/test/03-encrypt.t
+#[test]
+fn engine_test() {
+    use block_modes::{BlockMode, Cbc, block_padding::Pkcs7};
+
+    type Cipher = Cbc<magma::Gost89CryptoProA, Pkcs7>;
+
+    let key: [u8; 32] = [
+        0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF,
+        0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF,
+        0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF,
+        0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF,
+    ];
+    let iv: [u8; 8] = [0; 8];
+    let plaintext = b"The quick brown fox jumps over the lazy dog\n";
+
+    let c = Cipher::new_var(&key, &iv).unwrap();
+    let ct = c.encrypt_vec(plaintext);
+    for b in ct {
+        print!("0x{:02X}, ", b);
+    }
+    println!();
+}
+*/
```

Comment:
Strange, it seems rustfmt just wants to delete this entire block.

Perhaps use line comments instead of block comments? Or I wonder if there's an option to shut it off...

Reply:
Here's an issue about it...

https://github.com/rust-lang/rustfmt/issues/4028

Reply:
Yeah, certainly an annoying bug...